src/setupJest.ts

```diff
@@ -16,5 +16,5 @@ limitations under the License.
 
 SPDX-License-Identifier: Apache-2.0
 */
-// import 'jest-preset-angular/setup-jest';
+import 'jest-preset-angular/setup-jest'; // This allows jest tests to run in Visual Studio Code
```

Comment:
This I'm confused by - but not enough to block the PR...

I remember having to comment this out once because I think the `jest-preset-angular` setup ended up being called twice for some reason, related to the Angular dev builders in `angular.json` file. Tests still run though now, so why was this added back in? (For curiosity only)

Reply:
I installed "Jest Test Explorer" into Visual Studio Code to enable me to easily step through a test to debug it. The test explorer was unable to run the jest tests until I restored that line of code. Obviously my comment was not as clear as it should have been.